### PR TITLE
chore(deps): Bump eslint-config-sentry-app

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10578,35 +10571,6 @@ node-int64@^0.4.0:
   resolved "https://registry.yarnpkg.com/node-int64/-/node-int64-0.4.0.tgz#87a9065cdb355d3182d8f94ce11188b825c68a3b"
   integrity sha1-h6kGXNs1XTGC2PlM4RGIuCXGijs=
 
-"node-libs-browser@^1.0.0 || ^2.0.0":
```
Comment (billyvg) [inline yarn.lock:10581]: :))))

**Review by billyvg — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -3852,6 +3852,17 @@ array-includes@^3.0.3, array-includes@^3.1.1:
     es-abstract "^1.17.0"
     is-string "^1.0.5"
 
+array-includes@^3.1.3:
```
Comment (EvanPurkhiser) [inline yarn.lock:3855]: This is annoying

File: package.json [hunk 3]
```diff
@@ -149,6 +149,7 @@
     "@storybook/theming": "^6.2.9",
     "@visual-snapshot/jest": "^3.0.0",
     "@wojtekmaj/enzyme-adapter-react-17": "0.6.1",
+    "assert": "^2.0.0",
```
Comment (EvanPurkhiser) [inline package.json:152]: Storybook requires assert. See https://gist.github.com/shilman/8856ea1786dcd247139b47b270912324#gistcomment-3681971

We were previously implicitly relying on this

Comment (EvanPurkhiser) [reply package.json]: nvm just set it to false

**Pull request merged**